packages/react-charting/src/components/HeatMapChart/HeatMapChartRTL.test..tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { queryAllByAttribute, render, waitFor } from '@testing-library/react';
+import { HeatMapChart } from './index';
+import { HeatMapData } from './HeatMapChart.test';
+
+describe('HeatMap chart rendering', () => {
+  test('Should re-render the HeatMap chart with data', async () => {
+    // Arrange
+    const { container, rerender } = render(
+      <HeatMapChart
+        data={[]}
+        domainValuesForColorScale={[0, 600]}
+        rangeValuesForColorScale={['lightblue', 'darkblue']}
+      />,
+    );
+    const getById = queryAllByAttribute.bind(null, 'id');
+    // Assert
+    expect(container).toMatchSnapshot();
```

Comment:
I see snapshot file is missing for this test case.

Reply:
fixed the naming typo